knip.jsonc

```diff
@@ -55,7 +55,9 @@
     "@sentry/esbuild-plugin",
     "@liam-hq/cli",
     "@liam-hq/configs",
-    "@turbo/gen"
+    "@turbo/gen",
+    "langfuse",
+    "langfuse-vercel"
```

Comment:
(aside)This diff should not be needed after this PR is merged, and will be handled as soon as it is merged.
- https://github.com/liam-hq/liam/pull/1856

Reply:
(aside) fix it: https://github.com/liam-hq/liam/pull/1855/commits/c62efb423f045156e0fe7d9fea09327eb9f87696